app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @user_posts = Post.where(author_id: params[:user_id]).order(created_at: :desc).limit(5)
```

Comment:
- [ ] Same issue here. Kindly use the `recent_comments` method found in the post model to display the comments :+1: